index.d.ts

```diff
@@ -464,8 +464,2262 @@ interface Typography {
+      }
+      appearances: {}
+      sizes: {}
+    }
+    TableRow: {
+      baseStyle: {
+        outline: string
+        textDecoration: string
+        height: number
+        _isSelectable: { cursor: string }
+      }
+      appearances: {
+        default: { _hover: {}; _focus: {}; _active: {}; _current: {} }
+      }
+    }
+    TagInput: {
+      baseStyle: {
+        paddingY: string
+        backgroundColor: string
+        borderRadius: string
+      }
+      appearances: {
+        default: {
+          _focused: {
+            outline: string
+            zIndex: string
+            transition: string
+            boxShadow: string
+          }
+          _disabled: {
+            cursor: string
+            backgroundColor: string
+          }
+        }
+      }
+      sizes: {}
+    }
+    Text: {
+      baseStyle: {}
+      appearances: {}
+      sizes: {
+        300: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        400: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        500: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        600: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        small: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        medium: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+        large: {
+          fontSize: string
+          fontWeight: string
+          lineHeight: string
+          letterSpacing: string
+        }
+      }
+    }
+    TextDropdownButton: {
+      baseStyle: {
+        fontFamily: string
+        backgroundColor: string
+        borderRadius: string
+        paddingX: number
+        marginX: -4
+        paddingY: number
+        marginY: -2
+        color: string
+        _disabled: { cursor: string; pointerEvents: string }
+        _focus: { boxShadow: string }
+      }
+      appearances: {}
+      sizes: {
+        small: { fontSize: string; lineHeight: string }
+        medium: { fontSize: string; lineHeight: string }
+        large: { fontSize: string; lineHeight: string }
+      }
+    }
+    Tooltip: {
+      baseStyle: {
+        paddingY: number
+        paddingX: number
+        maxWidth: number
+        borderRadius: string
+        elevation: string
+      }
+      appearances: {
+        card: { backgroundColor: string }
+        default: { color: string; backgroundColor: string }
+      }
+      sizes: {}
+    }
+  }
+}
+
+export const defaultTheme: DefaultTheme
+
+export interface ClassicTheme extends Theme {
```

Comment:
I added this just for parity, but I'm not sure how many consumers are leveraging the type definitions + still on the classic theme. I'd imagine there's not many, so I'm happy to remove this too.